collections/Lyell/MS_Lyell_50.xml

```diff
@@ -48,25 +42,38 @@
                   <repository>Bodleian Library</repository>
                   <idno type="shelfmark">MS. Lyell 50</idno>
                </msIdentifier>
+               <head>Theological <foreign>quaestiones</foreign>; Gilbert Crispin; Honorius Augustodunensis</head>
                <physDesc>
-                  <p n="composite">Composite: fols. 1-40 || fols. 41-63</p>
+                  <p n="composite">Two MSS. written in Austria or Germany, bound together.</p>
+                  <objectDesc form="codex">
+                     <supportDesc material="perg">
+                        <support>parchment</support>
+                        <extent>63 leaves 
+                              <dimensions unit="mm" type="leaf"><height>220</height><width>150</width></dimensions>
+                           Edges cropped by binder.
+                        </extent>
+                        <foliation>Modern foliation.</foliation>
+                        <collation/>
+                     </supportDesc>
+                  </objectDesc>
+                  <bindingDesc>
+                     <p>Archaizing binding of second half of 15th cent. (not later than 1488), in wooden boards covered with rough white skin, with parchment pastedowns and original place marker of string. Original central leather strap and metal clasp fastening to pin on the upper cover – compare the clasp of MS. Lat. misc. d. 68, also from Admont. On the front cover is a parchment label with the title (15th cent.) <quote>Lucidarius de peccato originali</quote>.</p>
+                  </bindingDesc>
                </physDesc>
                <history>
-                  <provenance>
-                     <orgName key="org_128466393">Admont</orgName>, Benedictine monastery</provenance>
-               <provenance notBefore="1871" notAfter="1948" resp="#MMM"><persName role="fmo" key="person_64157046">James P. R. Lyell, 1871-1948</persName></provenance><acquisition when="1948" resp="#MMM">Chosen as one of the hundred manuscripts bequeathed to the Bodleian by Lyell in 1948.</acquisition></history>
+                  <provenance><orgName key="org_128466393">Benedictine monastery of Admont</orgName> in Austria: <quote>no. 563</quote> on spine of binding and inside front cover remains of MS. description (only surviving words: N° 563) as found in other Admont MSS. Listed in the 1380 catalogue: <quote>Item questio cuiusdam de originali peccato, incipit <quote>Queritur</quote>; in eodem lucidarius</quote>; see G. Möser-Mersky, <title>Mittelalt. Bibl.kat. Österreichs</title> iii (1961), p. 56 l. 14–15.</provenance>
+                  <provenance notBefore="1871" notAfter="1948" resp="#MMM">
```

Comment:
Not especially important, but the dating attributes here could be updated in the light of de la Mare's information